src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -163,7 +164,15 @@ method getChatColor*(self: Module): string =
   return chatDto.color
 
 method amIChatAdmin*(self: Module): bool =
-  return false
+  if(not self.controller.belongsToCommunity()):
+    let chatDto = self.controller.getChatDetails()
+    for m in chatDto.members:
+      if (m.id == singletonInstance.userProfile.getPubKey() and m.admin):
```

Comment:
I think the admin is always part of the members, so the check might be redundant here, but it's not a big deal

Reply:
@jrainville not sure what is redundant here?
So `chatDto.members` contains list of users with properties `id` and `admin`, myself as logged in user may be in a members list but `admin` flag for me may be either `true` or `false`, that means I am an admin of this chat if `m.id` is set to my public key and `m.admin` is set to `true`.

Reply:
Oops, my bad, I misread. I had not realized that `m` came from the loop. Ignore my comment